src/FSharpSource.targets

```diff
@@ -683,6 +719,44 @@
   </Choose>
   <Import Condition="'$(ProjectLanguage)' == 'FSharp'" Project="$(FSharpTargetsPath)"/>
 
+  <!-- Hook up DNX to enable solution refresh of packages -->
+  <PropertyGroup Condition="'$(TargetFramework)'=='coreclr'">
+    
+    <NuGetPackagesPath Condition="'$(NuGetPackagesPath)' == ''">$(FSharpSourcesRoot)\..\packages</NuGetPackagesPath>
+
+    <!-- fix this to be not hacky -->
```

Comment:
This enables nuget package restore for packages that require a version 3.0 nuget client to work.
nuget.exe is not a version 3.0 nugget client at this time.

So that we can do stuff like this:
msbuild src\fsharp\FSharp.Core\FSharp.Core.fsproj /p:TargetFramework=coreclr /t:Rebuild /p:RestorePackages=true